tools/gen_translations.py

```diff
@@ -161,7 +162,7 @@ def build_um_cf_map(fuseki, filename, now, git_sha):
         fh.writelines(cffc.lines(fuseki))
 
 
-def build_grib_cf_map(fuseki, filename, now, git_sha):
+def build_grib_cf_map(fuseki, now, git_sha, base_dir):
```

Comment:
A very minor point but I would prefer if `build_grib_cf_map` and `build_um_cf_map` had a similar argument order. Currently it's:

> def build_grib_cf_map(fuseki, now, git_sha, ***base_dir***):
> def build_um_cf_map(fuseki, ***filename***, now, git_sha):


The docstring of `build_grib_cf_map` also needs updating since `filename` is no longer an argument.